applications/GeoMechanicsApplication/tests/test_interface.py

```diff
@@ -30,8 +30,7 @@ def test_interface_side_cohesive(self):
 
         :return:
         """
-        test_name = 'test_interface_side_cohesive'
-        file_path = test_helper.get_file_path(os.path.join('.','test_interfaces', test_name + '.gid'))
+        file_path = test_helper.get_file_path(os.path.join('.','Interface', 'interface_side_cohesive'))
```

Comment:
Since you're constructing a relative path, I don't think you need the explicit `'.'`:
```suggestion
        file_path = test_helper.get_file_path(os.path.join('Interface', 'interface_side_cohesive'))
```

This suggestion also applies to the other relative paths in this file.

Reply:
fixed.